UXO_CF_3.tex

```diff
@@ -0,0 +1,319 @@
+    \item \daq{}: Communicate the supply cylinder pressure as read by the Fill Pressure Transducer.
+    \item \daq{}: Confirm that the two pressure measurements are in agreement.
+    \item \primary{} and \secondary: Retreat to the test control area, behind the blast shield.
+    \item \primary{}: Perform the following control system checks:
+    \begin{checklist}
+        \item Confirm that all actuator controls are in the ``off'' position:
+        \begin{checklist}
+            \item Remote Fill Valve
+            \item Tank Vent Valve
+            \item Injector Valve
+        \end{checklist}
+    \end{checklist}
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \ops{}: Poll the following personnel for GO/NO GO status:
+    \begin{checklist}
+        \item \peri{}
+        \item \perii{}
+        \item \heat
+        \item \daq{}
+        \item \primary{}
+        \item \secondary
+    \end{checklist}
+    \item \secondary: Give the system control key to \primary{}.
+    \item \primary{}: Engage the key switch and power on the control boxes.
+    \item \primary{}: Open the Tank Vent Valve.
+    \item \primary{}: Open the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \primary{}: Close the Remote Fill Valve.
+            \item \primary{}: Open the Injector Valve.
+            \item \secondary: Proceed only when the oxidizer tank has fully vented.
+            \item \primary{} and \secondary: Approach the test plumbing.
+            \item \primary{}: Close the Cylinder Valve.
+            \item \primary{}: Open the Remote Fill Valve.
+            \item \daq{}: Confirm that the Fill Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit plumbing setup.
+        \end{checklist}
+        \item If the Remote Fill Valve fails to open:
+        \begin{checklist}
+            \item \ops{}: Proceed to the Manual Control procedure.
+        \end{checklist}
+    \end{checklist}
+    \item \secondary{}: Proceed only when a white plume is visible from the Tank Vent Valve.
+    \item \primary{}: Close the Tank Vent Valve.
+    \item \primary{}: Close the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If the Remote Fill Valve fails to close:
+        \begin{checklist}
+            \item \primary{} and \secondary: Approach the test plumbing.
+            \item \primary{}: Close the Series Fill Valve.
+            \item \primary{} and \secondary: Retreat to the test control area, behind the blast shield.
+        \end{checklist}
+    \end{checklist}
+    \item \heat: Open the Tank Heating Valve.
+    \item \daq{}: Proceed only when the oxidizer tank pressure is at least 500 psi.
+    \begin{checklist}[label=$\bullet$]
+        \item If the oxidizer tank pressure does not reach 500 psi:
+        \begin{checklist}

```

Comment:
Ops look good, no real changes necessary, but is there any difference between what we do if the ox tank does reach 500 psi and what we do if it doesn't? It seems like we perform the test either way, but if we weren't at 500psi, we just review plumbing and test again. Is this accurate?

Reply:
Basically; we vent through the injector anyways because that's the fastest way to dump and there's not really a downside to doing so.

Reply:
Makes sense. Is the data worth capturing? Should we put in a line about "DAQ tech, make sure to save that data"?

Reply:
All data is logged while the labview program is running. If you mean "make sure you don't throw out that file," there isn't really anything to do for that until after the test is over.